fendermint/storage/src/im.rs

```diff
@@ -152,6 +155,23 @@ where
         }
         Ok(None)
     }
+
+    fn iterate<K, V>(&self, ns: &S::Namespace) -> impl Iterator<Item = KVResult<(K, V)>>
+    where
+        S: Decode<K> + Decode<V>,
+        <S as KVStore>::Repr: Ord + 'static,
+        K: 'static,
+        V: 'static,
```

Comment:
This needs to be static so K/V is guaranteed to outlive the iterator?

Reply:
I'm not sure I can explain, but the compiler insisted that the iterator can outlive these types :shrug: